sdk.d.ts

```diff
@@ -63,9 +63,9 @@ export type IconData = {
 /* The next code is autogenerated from sdk.mjs */
 /* eslint-disable */
 
-export const URL_REGEX: RegExp;
 export const SVG_PATH_REGEX: RegExp;
 export function getDirnameFromImportMeta(importMetaUrl: string): string;
+export function urlRegex(jsonschemaPath?: string): Promise<RegExp>;
```

Comment:
We used a `fs.readFile` for the JSON schema file. This means the `.jsonschema.json` file will be missed during the tree-shaking.

This means that our `urlRegex()` cannot be used in some scenarios.

Reply:
I've added `!.jsonschema.json` to `.npmignore`, so will be included in the package. This is the same behaviour of other functions like `getIconsData`, `getThisPartyExtensions` or `getThirdPartyLibraries`, which read from the icons data file and the README.